tests/commands/test_invoke_labelanalysis.py

```diff
@@ -203,7 +205,7 @@ def test_invoke_label_analysis(self, get_labelanalysis_deps, mocker):
             cli_runner = CliRunner()
             result = cli_runner.invoke(
                 cli,
-                ["-v", "label-analysis", "--token=STATIC_TOKEN", "--base-sha=BASE_SHA"],
+                ["label-analysis", "--token=STATIC_TOKEN", "--base-sha=BASE_SHA"],
```

Comment:
Just so I understand here...

Calling `-v` would configure the global logger to be verbose and then the rest of the tests would be run in verbose mode as well?  Whereas using the fixture just configures the logger for this test?

Reply:
Pretty much, yes. I don't know exactly how that works, but I'm sure there was some overlapping across the invokes and the other unit tests... some residual configuration from them was left in the env and it would log in verbose mode when it shouldn't.

But the fixture configures just that test and then returns it to the default level (INFO). So that test can run in verbose mode in an isolated fashion.